flow360/component/surface_mesh.py

```diff
@@ -57,7 +57,78 @@ def to_surface_mesh(self) -> SurfaceMesh:
         return SurfaceMesh(self.id)
 
 
-class SurfaceMeshDraft(ResourceDraft):
+class SurfaceMeshDraftFromSurfaceMesh(ResourceDraft):
```

Comment:
can you follow VolumeMeshDraft design so we don't have 2 separate classes?

The interface should be:

create surface mesh
```python
surface_mesh = fl.SurfaceMesh.create(
    Airplane.geometry, params=params, name="airplane-new-python-client"
)
```


upload surface mesh
```python
surface_mesh = fl.SurfaceMesh.from_file(
    "file.stl", name="airplane-new-python-client"
)
```

both should return `SurfaceMeshDraft` object which can be submitted:

```python
surface_mesh = surface_mesh.submit()
```

Reply:
fixed.